cram_knowrob/cram_cloud_logger/src/cloud-logger-client.lisp

```diff
@@ -135,6 +135,9 @@
 (defun export-log-to-owl (filename)
   (send-prolog-query-1 (create-query "rdf_save" (list (concatenate 'string "\\'/home/ros/user_data/" filename "\\'" ) "[graph(\\'LoggingGraph\\')]"))))
 
+(defun export-belief-state-to-owl (filename)
+  (json-prolog:prolog-simple-1 (concatenate 'string "rdf_save(" (concatenate 'string "'/home/ros/user_data/" filename "'" "," "[graph('belief-state')])"))))
```

Comment:
Could you please give a default filename for the function?

(defun blabla (&optional (filename "blabla"))